### PR TITLE
borg: add page

### PR DESCRIPTION
**Pull request by ckovsky**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by agnivade** [comment]
@ckovsky - Please sign the CLA. We can get on with the review after that.

File: pages/common/borg.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# borg
+
+> Deduplicating backup tool with compression and encryption.
+> Supports local and remote backups, mountable as filesystems.
```
Comment (sbrl) [inline pages/common/borg.md]: Does this mean that the remote backups have to be mounted as a file system, or that backs can be mounted as a file system?

File: pages/common/borg.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Initialise a (local) repository:
+
+`borg init {{/path/to/repo/folder}}`
+
+- Backup a folder into the repository, creating an archive called Monday:
+
+`borg create --progress {{/path/to/repo/folder::Monday}} {{/path/to/source/folder}}`
```
Comment (sbrl) [inline pages/common/borg.md]: Are the `::` required here? Perhaps we could tweak this to `{{/path/to/repo/folder}}::{{Monday}}` if that's the case?

File: pages/common/borg.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- List all archives in a repository:
+
+`borg list {{/path/to/repo/folder}}`
+
+- Extract a specific folder from the 'Monday' archive, excluding all *.ext files:
+
+`borg extract {{/path/to/repo/folder::Monday}} {{path/to/target/folder}} --exclude '*.ext'`
```
Comment (sbrl) [inline pages/common/borg.md]: Perhaps you could put `{{` and `}}` around `*.ext`? Also, is the `'` meant to be there?

File: pages/common/borg.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Prune a repository, deleting all archives older than 7 days, listing changes:
```
Comment (sbrl) [inline pages/common/borg.md]: Perhaps `Prune a repository by deleting`.... would read better?

File: pages/common/borg.md (hunk 1 above)
Comment (ckovsky) [reply pages/common/borg.md]: It's the backups that can be mountable. But borg can backup remotely over ssh. The command for an init would look like:
$ borg init user@hostname:backup

I wasn't sure whether to include a remote example or not. Wanted to keep it simple :-)

File: pages/common/borg.md (hunk 2 above)
Comment (ckovsky) [reply pages/common/borg.md]: Yeah, the :: are part of the syntax. I like the suggestion of separating the archive name.

File: pages/common/borg.md (hunk 4 above)
Comment (ckovsky) [reply pages/common/borg.md]: Sounds good!

File: pages/common/borg.md (hunk 3 above)
Comment (ckovsky) [reply pages/common/borg.md]: The `'` are required, as far as I know. Yes, might be more consistent to add `{{` and `}}`.

File: pages/common/borg.md (hunk 1 above)
Comment (ckovsky) [reply pages/common/borg.md]: Maybe the description could be simplified. It sounds a bit like an ad...

File: pages/common/borg.md (hunk 3 above)
Comment (sbrl) [reply pages/common/borg.md]: Ah, I see. Thanks!

File: pages/common/borg.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# borg
+
+> Deduplicating backup tool with compression and encryption.
+> Supports local and remote backups that are mountable as filesystems.
```
Comment (agnivade) [inline pages/common/borg.md]: I don't see an example which works on remote backups. Perhaps you can switch one of the examples to work on a remote backup.

File: pages/common/borg.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Initialise a (local) repository:
+
+`borg init {{/path/to/repo/folder}}`
```
Comment (agnivade) [inline pages/common/borg.md]: Please change it to `path/to/repo_folder`. Apply this convention everywhere.

File: pages/common/borg.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Backup a folder into the repository, creating an archive called Monday:
```
Comment (agnivade) [inline pages/common/borg.md]: Monday in quotes ("")

File: pages/common/borg.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`borg create --progress {{/path/to/repo/folder}}::{{Monday}} {{/path/to/source/folder}}`
+
+- List all archives in a repository:
+
+`borg list {{/path/to/repo/folder}}`
+
+- Extract a specific folder from the 'Monday' archive, excluding all *.ext files:
```
Comment (agnivade) [inline pages/common/borg.md]: Monday in double quotes.

File: pages/common/borg.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`borg extract {{/path/to/repo/folder}}::{{Monday}} {{path/to/target/folder}} --exclude '{{*.ext}}'`
+
+- Prune a repository by deleting all archives older than 7 days, listing changes:
+
+`borg prune --keep-within 7d --list {{/path/to/repo/folder}}`
```
Comment (agnivade) [inline pages/common/borg.md]: 7d within tokens {}

**Pull request merged**